iDiff/differs/pipDiff.go

```diff
@@ -64,18 +66,28 @@ func (d PipDiffer) getPackages(path string) (map[string]utils.PackageInfo, error
 			if len(packageMatch) != 0 {
 				packageName := packageMatch[1]
 				version := packageMatch[2][:len(packageMatch[2])-1]
-				size := strconv.FormatInt(c.Size(), 10)
-				packages[packageName] = utils.PackageInfo{version, size}
 
-				continue
-			}
+				// Retrieves size for actual package/script corresponding to each dist-info metadata directory
+				// by taking the file entry alphabetically before it (for a package) or after it (for a script)
```

Comment:
pip creates a folder called "NAME-VERSION.dist-info" for every package installed, and since they're sorted alphabetically, once we find a directory entry matching that regex pattern (because there are some other non-package directories there too that we want to ignore), we know that there must be a package called NAME, and since we're iterating through the contents alphabetically, it would come directly before this dist-info folder (unless there's no folder for it, just a NAME.py script, in which case it would come immediately after its dist-info folder, which is the other case here)